### ISSUE TITLE
Need to update the link to JSR 352 spec project in README.md

### ISSUE THREAD

**Issue by chengfang** [jberet/jsr352#106]
since JSR-352 spec project has migrated out of java.net, we should update JBeret project README.md to use the current location.

It can be either the spec page at jcp.org (https://jcp.org/en/jsr/detail?id=352), or the RI project page at github (https://github.com/WASdev/standards.jsr352.jbatch/).

**Comment by rwperrott** [comment]
Indeed, still not updated, and flagged by as **insecure** by Brave!  Not a promising intro.
